stats/type_of_matrix.g

```diff
@@ -0,0 +1 @@
+isParallel := true;; dimension :=5;; rank := 1;; ring := GF(2);; numberChops := 1;;
```

Comment:
This file should not have been committed.

Reply:
I thought about that and decided that I would commit it because `stats/calculate_time.g` uses it and I would like to have this program working from the scratch. If you don't think so and it is easy for you to do then just remove it.